base/doc/usrguide.tex

```diff
@@ -837,10 +837,12 @@ \subsection{Using the verbatim argument types}
 
 Some additional details that may be useful for those with more \TeX{}
 knowledge: do not worry if this does not make sense to you! Spaces and tabs are
-stored as active characters. In Unicode engines, all other characters are of
-type \enquote{other}. In $8$-bit engines, the ASCII characters other than tab
-and space are of type \enquote{other}, and non-ASCII characters are active. As
-such, token-based comparisons are likely to fail unless set up properly.
+stored as active characters. In $8$-bit engines and non-ASCII characters are
+active, whilst other than the letters a--zA--Z, ASCII characters are
+\enquote{other}. In Unicode engines, non-ASCII codepoints will be either
+letters or \enquote{other}, based on the standard \LaTeX{} settings derived
+from Unicode data. As such, token-based comparisons are likely to fail unless
+set up properly.
```

Comment:
not sure I understand this makes a likely fail

Reply:
I've revised here: with letters and 'other', the only issue really would be the spaces/tabs.

Reply:
I had read this as being related to the Unicode engines which is obviously isn't. Perhaps start a new paragraph and explicitly mention spaces/tabs?